hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1691,6 +1694,18 @@ message DeleteSnapshotRequest {
   optional uint64 deletionTime = 4;
 }
 
+message SnapshotMoveDeletedKeysRequest {
+  required SnapshotInfo fromSnapshot = 1;
+  optional SnapshotInfo nextSnapshot = 2;
+  repeated KeyValuePair activeDBKeys = 3;
+  repeated KeyValuePair nextDBKeys = 4;
+}
+
+message KeyValuePair {
```

Comment:
`KeyValuePair` sounds way too generic. Name more specifically according to its usage.

Even better, put a comment around its intended usage.